internal/kubectl/root.go

```diff
@@ -62,6 +62,10 @@ func init() {
 	client = &realClient{}
 
 	kubecfgFlags := genericclioptions.NewConfigFlags(false)
+	defaultNs, _, ise := kubecfgFlags.ToRawKubeConfigLoader().Namespace()
```

Comment:
Can you update `ise` to `err` to follow the pattern of the rest of the repo.

Reply:
yes, rebased.